### ISSUE TITLE
Track down output issue

### ISSUE THREAD

**Issue by sdras** [sdras/array-explorer#13]
Three people on Chrome on a Mac had an issue with the output not showing up. I think it has to do with the timing of the scale in gsap. Need to investigate.

**Comment by Ana-Belen** [comment]
I've just had this issue. 
The content is actually there, but it has a scale:  0 applied to it, that make it invisible. I deactivated that style from my dev tools and it was fine. 

This is a super useful tool. Thank you!

**Comment by chrisVillanueva** [comment]
I tested the app on localhost -- which works great!  I did not encounter the "output section not showing up" issue.  Is there additional information about this bug?  Here are my system specs:

**My device:**

- laptop: MacBookPro
- os: macOS High Sierra ver. 10.13.2
- chrome browser: Version 63.0.3239.84 (Official Build) (64-bit)

Here's a screenshot of the application on localhost:

![screen shot 2017-12-29 at 12 10 51 am](https://user-images.githubusercontent.com/1554280/34430690-4c7ea67c-ec2d-11e7-9a02-f6e2d62fe9e5.png)
